test/powershell/Host/PSVersionTable.Tests.ps1

```diff
@@ -18,7 +18,9 @@ Describe "PSVersionTable" -Tags "CI" {
 
     }
     It "GitCommitId property should not contain an error" {
-       $PSVersionTable.GitCommitId | Should not match "powershell.version"
+       $powershellVersionFile = Join-Path -Path $PSScriptRoot -ChildPath "assets/powershell.version"
```

Comment:
I tried to do the test with direct checking. It seems indirect validation will catch a bug too.

Fixed.